website/integrations/services/paperless-ngx/index.md

```diff
@@ -35,33 +35,52 @@ Create an application (under Resources/Applications) with these settings:
     Provider: Paperless
 
 ## Paperless Configuration
-
-Add the following environment variables to your Paperless-ngx setup. If you are using Docker Compose, then add the following to your docker-compose.yml file:
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs
+  defaultValue="docker"
+  values={[
+    {label: 'Docker', value: 'docker'},
+    {label: 'Standalone', value: 'standalone'},
+  ]}>
+  <TabItem value="docker">
+If you have Paperless-ngx setup in docker, add the following environment variables for authentik
 
 ```yaml
-PAPERLESS_APPS: allauth.socialaccount.providers.openid_connect
-PAPERLESS_SOCIALACCOUNT_PROVIDERS: >
-    {
-      "openid_connect": {
-        "APPS": [
-          {
-            "provider_id": "authentik",
-            "name": "Authentik",
-            "client_id": "< Client ID >",
-            "secret": "< Client Secret >",
-            "settings": {
-              "server_url": "https://authentik.company/application/o/paperless/.well-known/openid-configuration"
-            }
+environment:
+    PAPERLESS_APPS: allauth.socialaccount.providers.openid_connect
+    PAPERLESS_SOCIALACCOUNT_PROVIDERS: >
+        {
+          "openid_connect": {
+            "APPS": [
+              {
+                "provider_id": "authentik",
+                "name": "Authentik",
+                "client_id": "<Client ID>",
+                "secret": "<Client Secret>",
+                "settings": {
+                  "server_url": "https://authentik.company/application/o/paperless/.well-known/openid-configuration"
+                }
+              }
+            ],
+            "OAUTH_PKCE_ENABLED": "True"
           }
-        ],
-        "OAUTH_PKCE_ENABLED": "True"
-      }
-    }
+        }
 ```
-
 Now restart your container:
 `docker compose down && docker compose up -d`
+  </TabItem>
+  <TabItem value="standalone">
 
+edit your `paperless.conf` and add the following:
```

Comment:
The problem is that it could be located in four different locations.
I just expected the user to know where their config is if they installed it Bare Metal.

But sure, I guess it doesn't hurt to add the possible locations.

```suggestion
You need to update your `paperless.conf` configuration file. Paperless will search for this configuration file in the following locations and use the first one it finds:
- The environment variable `PAPERLESS_CONFIGURATION_PATH`
- `/path/to/paperless/paperless.conf`
- `/etc/paperless.conf`
- `/usr/local/etc/paperless.conf`

Edit your `paperless.conf` and add the following:
```